extractor/spotify/core.py

```diff
@@ -1,14 +1,31 @@
-from spotipy import Spotify
-from spotipy.oauth2 import SpotifyClientCredentials
-from dotenv import load_dotenv, find_dotenv
-import urllib.parse as parser
+from extractor.spotify.client import spotify_client
 
-load_dotenv(find_dotenv())
-# env variables: SPOTIPY_CLIENT_ID, SPOTIPY_CLIENT_SECRET
+class Track:
+    def __init__(self, id):
```

Comment:
The class constructor has an id field, but the class definition seems to be missing it. Is this intended?